astro/src/content/docs/release-notes/index.mdx

```diff
@@ -44,6 +44,104 @@
 
 Looking for release notes older than 1.44.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.55.0' releaseDate='December 18, 2024' name='Holiday Hippo'/>
+
+<DatabaseMigrationWarning />
+
+### Security
+* <Issue issue="2906">
+  Added additional validation checks for authorized URLs in OAuth workflows
+  </Issue>
+* <Issue issue="2916">
+  Add additional validation of an authorizing JWT when issuing a new JWT.
+  </Issue>
+
+### Changed
+* As part of the new refresh token grace period functionality, the length of the FusionAuth refresh token is increasing by a small amount. You'll need to accommodate for this if you are storing refresh tokens using storage that is expecting tokens to always exist at the previous tokens' length.
+  * Related to [GitHub Issue #2952](https://github.com/FusionAuth/fusionauth-issues/issues/2952)
+
+### New
+* <Issue issue="2741">
+  Allow for the sending of usage stats. Usage stats allow us to understand how our users use our product, which helps us know where we need to invest in new features and enhancements. 
```

Comment:
same question I had on Brent's PR, is `usage stats` what we want to use for customer-facing text? As opposed to something like `usage statistics`

Reply:
We're using `usage stats` in all of our marketing content. I can bring it up for discussion, but will leave it consistent for now.